knip.json

```diff
@@ -1,15 +1,17 @@
 {
     "$schema": "https://unpkg.com/knip@5/schema.json",
+    "ignore": ["generated/**/*.ts"],
```

Comment:
Let's add comment why we are commenting out knip.json.
Also check if we can use jsonc format for knip